docs/source/user-guide/tasks/manage-environments.rst

```diff
@@ -337,18 +337,50 @@ the environment, and running any activation scripts that the environment may
 contain. These activation scripts are how packages can set arbitrary
 environment variables that may be necessary for their operation.
 
+When `installing Anaconda <http://docs.continuum.io/anaconda/install.html>`_,
+you have the option to “Add Anaconda
+to my PATH environment variable.” This is not recommended because the
+add to PATH option appends Anaconda to PATH. When the installer appends
+to PATH, it does not call the activation scripts.
+
+On Windows, PATH is composed of two parts, the system PATH and the
+user PATH. The system PATH always comes first. When you install
+Anaconda for Just Me, we add it to the user PATH. When you install
+for All Users, we add it to the system PATH. In the former case,
+you can end up with system PATH values taking precedence over
+our entries. In the latter case, you do not. We do not recommend
+All Users installs.
+
+Activation prepends to PATH. This only takes effect
+when you have the environment active so it is local to a terminal session,
+not global.
+
 To activate an environment: ``conda activate myenv``
 
 .. note::
    Replace ``myenv`` with the environment name or directory path.
 
 Conda prepends the path name ``myenv`` onto your system command.
 
+You may receive a warning message if you have not activated your environment:
+
+.. code-block:: Python
+
+   Warning:
+   This Python interpreter is in a conda environment, but the environment has
+   not been activated. Libraries may fail to load. To activate this environment
+   please see https://conda.io/activation.
+
+If you receive this warning, you need to activate your environment. To do
+so on Windows, run: ``c:\Anaconda3\scripts\activate base`` in
```

Comment:
Minor nit, Scripts should have a capital S (Windows has learnt case sensitivity and we should be using it and promoting it I think).